webapp/channels/src/components/advanced_text_editor/send_button/send_post_options/core_menu_options.tsx

```diff
@@ -23,6 +30,69 @@ function CoreMenuOptions({handleOnSelect, channelId}: Props) {
         isDM,
     } = useTimePostBoxIndicator(channelId);
 
+    const recentlyUsedCustomDate = useSelector((state: GlobalState) => getPreference(state, scheduledPosts.SCHEDULED_POSTS, scheduledPosts.RECENTLY_USED_CUSTOM_TIME));
+
+    interface RecentlyUsedCustomDate {
```

Comment:
You can declare this outside the component.

Reply:
same here, should we create a utils file for this module?

Reply:
done